src/raygun.network-tracking.js

```diff
@@ -217,9 +217,7 @@ window.raygunNetworkTrackingFactory = function(window, Raygun) {
                 executeHandlers();
               }
             })
-          );
-
-          promise.catch(
+          ).catch(
```

Comment:
We were able to replicate (As long a the error throwing code was referenced from an file (not an in line script)